docs/content/contributing.md

```diff
@@ -61,22 +61,34 @@ By default, the system tests run in a headless Chrome browser. Prefix the test c
 
 ## Writing documentation
 
-Documentation is written as [YARD](https://yardoc.org/) comments directly in the source code, compiled into Markdown via `rake docs:build` and served by [Doctocat](https://github.com/primer/doctocat).
+Documentation is written as [YARD](https://yardoc.org/) comments directly in the source code, compiled into Markdown via `build exec rake docs:build` and served by [Doctocat](https://github.com/primer/doctocat).
 
 ### Storybook / Documentation / Demo Rails App
 
+* **Storybook**: Components can be interacted with, seen in different contexts, and controls can be maniuplated
+  * Typically runs on port 5000
+  * To rebuild stories, navigate to the `demo` directory (`cd demo`) and then run `bin/rails view_component_storybook:write_stories_json`
+* **Docs**: Generated YARD docs with examples, see components with usage instructions and examples
+  * Typically runs on port 5400
+  * To rebuild docs, run `bundle exec rake docs:build`
+* **Demo App**: See components on a plain page with no interfering framework or styling. Used to test functionality.
+  * Typically runs on port 4000 - visit `/rails/view_components/` in your browser
+  * To rerender the templates, you do not have to restart the server. Run this command and refresh the page.
```

Comment:
Which command does this bullet point refer to?

Reply:
Whoops - thank you - updated! https://github.com/primer/view_components/pull/1016/commits/35f2be5867dc9ea7d5df7d8d46c9739e9c1ddb0e